apicomponents/service.py

```diff
@@ -0,0 +1,911 @@
+        try:
+            for service in range(0, len(response.json())):
+                servicelist.append(response.json()[service-1])
+            while len(response.json()) == pageSize:
+                pageNumber += 1
+                response = self.session.get(self.apicall + f"/v1/service/project/{uuid}", params={'pageSize': pageSize, 'pageNumber': pageNumber})
+                for service in range(0, len(response.json())):
+                    servicelist.append(response.json()[service-1])
+            if response.status_code == 200:
+                return servicelist
+        except :
+            if response.status_code == 404:
+                return (f"The project could not be found, {response.status_code}")
+            elif response.status_code == 403:
+                return (f"Access to the specified project is forbidden, {response.status_code}")
+            elif response.status_code == 401:
+                return (f"Unauthorized, {response.status_code}")
+            else:
+                return ((response.content).decode("utf-8"), response.status_code)
+            
+    def get_service(self,uuid):
+        """Returns a specific service.
+
+        Args:
+            uuid (string): The UUID of the project.
+
+        Returns:
+            dict: Returns a specific service.
+        """
+        response = self.session.get(self.apicall + f"/v1/service/{uuid}")
+        if response.status_code == 200:
+            return response.json()
+        elif response.status_code == 401:
+            return (f"Unauthorized, {response.status_code}")
+        elif response.status_code == 404:
+            return (f"The service could not be found, {response.status_code}")
+        elif response.status_code == 403:
+            return (f"Access to the specified project is forbidden, {response.status_code}")
+        else:
+            return ((response.content).decode("utf-8"), response.status_code)
+        
+    def delete_service(self, uuid):
+        """Deletes a service
+
+        Args:
+            uuid (string): The UUID of the project. 
+        """
+        response = self.session.delete(self.apicall + f"/v1/service/{uuid}")
+        if response.status_code == 200:
+            return (f"Successful operation")
+        elif response.status_code == 403:
+            return (f"Access to the specified service is forbidden, {response.status_code}")
+        elif response.status_code == 401:
+            return (f"Unauthorized, {response.status_code}")
+        elif response.status_code == 404:
+            return (f"The UUID of the service could not be found, {response.status_code}")
+        else:
+            return ((response.content).decode("utf-8"), response.status_code)
+        
+    def create_service(self, uuid, providerName=None, providerURL=None, contactName=None, contactEmail=None, contactPhone=None):
+        """
+        Model:
```

Comment:
@tonnyhideyori Is the Model listed in the swagger of DependencyTrack? If yes, please refer to it and remove this long multiline comment.

Reply:
ok